docs/lms_config.ini

```diff
@@ -32,10 +31,17 @@ progress = SFD_check_progress.html
 
 learneraccountsettings = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-account-settings-page
 
-learnerdashboard = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-learner-dashboard
+learnerdashboard = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-dashboard
+
+programs = SFD_enrolling.html#programs
 
 bookmarks = SFD_bookmarks.html
 
```

Comment:
I notice that the existing values seem to have a space before them, but the new ones don't. I have no idea whether this will make a difference in functionality, but figured I'd point it out just in case